frontend/language/src/nb.json

```diff
@@ -870,7 +869,11 @@
   "schema_editor.error_could_not_detect_taskType": "Kunne ikke hente oppgavetype for {{layout}}",
   "schema_editor.error_could_not_detect_taskType_description": "Dette gjør at vi ikke kan vise riktig liste over tilgjengelige komponenter. Velg en annen sidegruppe eller prøv å last siden på nytt.",
   "schema_editor.error_data_type_name_exists": "Modellen kan ikke ha samme navn som datatyper i løsningen.",
+  "schema_editor.error_depth": "Du kan ikke plassere gruppen her, fordi skjemaet får for mange nivåer.",
+  "schema_editor.error_invalid_child": "Du kan ikke plassere den komponenttypen i denne gruppen.",
+  "schema_editor.error_invalid_datamodel_name": "Navnet er ugyldig. Det første tegnet må være en bokstav (a-z). Deretter kan du bruke bokstaver (a-å), tall, understrek og bindestrek.",
   "schema_editor.error_model_name_exists": "Modellnavnet {{newModelName}} er allerede i bruk.",
+  "schema_editor.error_reserved_keyword": "Navnet er reservert og kan ikke brukes.",
```

Comment:
```suggestion
  "schema_editor.error_reserved_keyword": "Du kan ikke bruke dette navnet, det er allerede reservert.",
```

Reply:
Hva skjer når man reserverer et navn? Er det før det er lagt til på et vis? Av noen andre? Er det et utkast? Kna man ha to faner oppe samtidig, er det da det blir reservert?

Reply:
Dette er for navn som er reservert av programmeringsspråket C#. Datamodellgenerering vil feile hvis brukeren har noen av de reserverte navnene som modellnavn.

Reply:
Dette er navnene det gjelder:
![bilde](https://github.com/user-attachments/assets/a3422e4c-1d81-4d1f-9ba5-013ef476c2d7)